tools/hmaptool/README.md

```diff
@@ -0,0 +1,4 @@
+# hmaptool
+
+This is a tool which creates binary headermaps to enable angle bracket imports of headers when building with Bazel.
+Required to mimic Xcode's behavior of generating headermaps by default which iOS projects rely on.
```

Comment:
This is a general tool for `rules_apple` and not iOS specific, correct?
```suggestion
Required to mimic Xcode's behavior of generating headermaps by default which projects rely on.
```

Reply:
Made the change thanks!